src/c4group/CStdFile.h

```diff
@@ -20,7 +20,7 @@
 #ifndef INC_CSTDFILE
 #define INC_CSTDFILE
 
-#include <stdio.h>
+#include <cstdio>
```

Comment:
I don't think there's a single C++ compiler that implements <cstdio> by directly declaring the required members inside namespace std instead of just including <stdio.h> and `using` them into std, so this should be fine.